corehq/apps/app_manager/views.py

```diff
@@ -2049,6 +2050,8 @@ def edit_advanced_form_actions(request, domain, app_id, module_id, form_id):
     json_loads = json.loads(request.POST.get('actions'))
     actions = AdvancedFormActions.wrap(json_loads)
     form.actions = actions
+    if advanced_actions_use_usercase(form.actions) and not is_usercase_in_use(domain):
```

Comment:
the `is_usercase_in_user` check is redundant since that's already checked in `enable_usercase`